pkg/dsdk/connection.go

```diff
@@ -375,17 +382,16 @@ func (c *ApiConnection) do(ctxt context.Context, method, url string, ro *greq.Re
 		return c.retry(ctxt, method, url, ro, rs, sensitive)
 	}
 	if eresp != nil {
-		Log().Errorf("Recieved API Error %s", Pretty(eresp))
+		detailLog.Errorf("Recieved API Error %s", Pretty(eresp))
 		return eresp, nil
 	}
 	if err != nil {
-		Log().Errorf("Error during checkResponse: %s", err)
+		detailLog.Errorf("Error during checkResponse: %s", err)
```

Comment:
I missed this when I renamed checkResponse
```suggestion
		detailLog.Errorf("Error during translateErrors: %s", err)
```